src/cnf/mod.rs

```diff
@@ -25,14 +25,37 @@ pub const SERVER_AGENT: &str = concat!("SurrealDB ", env!("CARGO_PKG_VERSION"));
 #[cfg(feature = "has-storage")]
 pub const APP_ENDPOINT: &str = "https://surrealdb.com/app";
 
-/// How many concurrent tasks can be handled in a WebSocket
-#[cfg(feature = "has-storage")]
-pub const MAX_CONCURRENT_CALLS: usize = 24;
-
 /// Specifies the frequency with which ping messages should be sent to the client
 #[cfg(feature = "has-storage")]
 pub const WEBSOCKET_PING_FREQUENCY: Duration = Duration::from_secs(5);
 
+/// Set the maximum WebSocket frame size to 16mb
+#[cfg(feature = "has-storage")]
+pub static WEBSOCKET_MAX_FRAME_SIZE: Lazy<usize> = Lazy::new(|| {
+	let default = 64 << 20;
```

Comment:
64 << 20 = 67_108_864 = 4.19 larger than 16MB

Reply:
Good spot! I've changed this to `16MiB`